templates/firstbyte/base.html

```diff
@@ -21,7 +21,7 @@
     {% block ga %}{% if GA_TRACKING_ID %}{% include 'firstbyte/ga.html' %}{% endif %}{% endblock %}
 </head>
 <body class="d-flex flex-column">
-    <nav class="sticky-top navbar navbar-expand-lg navbar-dark bg-dark py-3">
+    <nav class="sticky-top navbar navbar-expand-lg bg-green py-3">
```

Comment:
Okay, I think I was able to get the desired output by reverting the changes to bootstrap.css while using `navbar-light bg-green`. I believe `navbar-dark` just changes default text color whereas `bg-green` sets the background. Please let me know if I'm misunderstanding the issue!